File: LeetcodeEasyProblems/_191. Number of 1 Bits/Solution.java
class Solution {
    public int hammingWeight(int n) {
        int count = 0;
        // Loop 32 times through the bit of n
        while (n != 0){
            // Check if the LSB of the current n
            // is 1 or not and count it
            if ((n & 1) == 1){
                count++;
            }
            n >>= 1;
        }
        return count;
        
    }
}

File: LeetcodeEasyProblems/_191. Number of 1 Bits/Solution.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        count = 0
        while n:
            #Shift the inital bit number to the right
            #i times so that its LSB got changed
            if (n & 1) == 1:
                count+= 1
            n = n >> 1
        
        return count


        


        


    
